### DISCUSSION TITLE
Samsung Monitor LS32AG55x detected but no longer changing brightness

### DISCUSSION THREAD

**Discussion by mindlessunicorn** [xanderfrangos/twinkle-tray#717] [Monitor Compatibility Issues]
So my Samsung monitor can be detected by TT, but recently the program can't actually change the brightness anymore like it used to. The brightness values on the program can be changed, but my monitor's brightness doesn't follow the changes. Is this a problem with the software or with my monitor?

**Comment by xanderfrangos** [comment]
It's usually either Twinkle Tray, the monitor, or the graphics drivers causing the issue. I recommend going through this list: https://github.com/xanderfrangos/twinkle-tray/wiki/Display-Detection-&-Support-Issues#common-reasons-for-monitors-not-being-supported-or-detected
